Index: libreelec-alsa-utils/src/packages/audio/alsa-utils/scripts/soundconfig
===================================================================
--- libreelec-alsa-utils.orig/src/packages/audio/alsa-utils/scripts/soundconfig
+++ libreelec-alsa-utils/src/packages/audio/alsa-utils/scripts/soundconfig
@@ -15,7 +15,7 @@ mixer() {
 (
 . /etc/profile
 
-progress "Setting up sound card"
+echo "Setting up sound card" | cat
 
 if [ -f $HOME/.config/sound.conf ]; then
 
